routers/web/repo/http.go

```diff
@@ -493,7 +493,12 @@ func serviceRPC(h serviceHandler, service string) {
 	var stderr bytes.Buffer
 	cmd := exec.CommandContext(ctx, git.GitExecutable, service, "--stateless-rpc", h.dir)
 	cmd.Dir = h.dir
+
 	cmd.Env = append(os.Environ(), h.environ...)
+	cargs := make([]string, len(git.GlobalCommandArgs))
+	copy(cargs, git.GlobalCommandArgs)
+	cmd.Env = append(cmd.Env, cargs...)
```

Comment:
You overload it! and drop old val!

Reply:
Oh you just need to change Env with Arhs

Reply:
Overload it?

What do you mean? Could you send a "```suggestion"?

Reply:
In few min yes